tools/debian/convert_debian.py

```diff
@@ -136,13 +141,13 @@ def dumper(obj):
 
 
 def parse_security_tracker_file(advisories: Advisories,
-                                security_tracker_repo: str):
+                                security_tracker_repo: str, lts: bool):
   """Parses the security tracker files into the advisories object"""
 
   codename_to_version = create_codename_to_version()
-
+  security_tracker_path = SECURITY_TRACKER_DLA_PATH if lts else SECURITY_TRACKER_DSA_PATH
```

Comment:
in general it's better to make functions like these more general and avoid hardcoding cases like these.

Let's have this function take in a string path instead and have the caller make the decision about which path to read from.